### PR TITLE
Rudimentary way to link to files.

### PR DESCRIPTION
**Pull request by Mogyula**
Hi!

This is my very first pull request sent to any project ever; I hope this is how it works. :)

So it was when I deployed a Digital Garden based on an Obsidian vault that references a ton of locally saved PDFs (as to support ideation for a reasearch project) that I realized that these wouldn't work online, as the plugin wouldn't commit ordinary files, only notes and images I guess.

Now I didn't do any work on the plugin, but I came to realize that if I just commit those PDFs manually, and also apply these changes to the codebase, then they'd be accessible.

The change on the regexp in .eleventy.js:160 is to cosnider links with or without the pipeline symbol, because I've noticed that notes' links get converted to the labeled format as in [[Another page|Another page]] even if in Obsidian it was just [[Another page]]. With PDFs though, you get [[dummy.pdf]] without the pipe.

See [this](https://digitalgarden-ten-weld.vercel.app/) site for a demo.

Gyula

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -157,46 +157,63 @@ module.exports = function (eleventyConfig) {
...
-            noteIcon = frontMatter.data.noteIcon;
-          }
-        } catch {
-          deadLink = true;
+        let noteIcon = process.env.NOTE_ICON_DEFAULT;
+        let permalink;
+        let headerLinkPath;
+        
+          
+        if (fileLink.slice(-5).includes(".") && fileLink.slice(-3)!=".md"){
```
Comment (oleeskild) [inline .eleventy.js:176]: Why does slice(-5).includes(".") imply its a file? This seems to only work with filetypes that have an extension that is smaller than 5 characters.

File: src/site/notes/Dummy page.md
Comment (oleeskild) [inline src/site/notes/Dummy page.md:1]: This file need to be removed before this PR can be merged.

File: src/site/etc/dummy.pdf
Comment (oleeskild) [inline src/site/etc/dummy.pdf:1]: This file need to be removed before this PR can be merged.

**Review by oleeskild — CHANGES_REQUESTED** [review]
Sorry for being so slow reviewing this. I've added some comments on things that needs to be changed for this to be able to be merged in.

**Comment by Mogyula** [comment]
Sorry on my part as well for getting back to this with such a delay. I believe the requested changes have been made. Let me know otherwise.

**Pull request closed without merging**